cookbooks/oceanic_extension/oceanic_extension.prm

```diff
@@ -0,0 +1,273 @@
+### Oceanic Extension Cookbook
+# A cookbook to illustrate the parameters and processes controlling 
+# non-linear deformation of oceanic lithosphere. This parameter file 
+# is the first in a series of cookbooks that systematically build in complexity, 
+# and thus serves as a reference model for the remaining cookbooks.
+
+#  Global parameters
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 2e6
+set Resume computation                     = auto
+set Use years in output instead of seconds = true
+set Nonlinear solver scheme                = single Advection, iterated defect correction Stokes
+set Nonlinear solver tolerance             = 1e-5
+
+# We set the number of nonlinear iterations to 100 as the first 1-2 time steps 
+# often require a large number of iterations to converge.
+# Subsequent time steps should require substantially fewer nonlinear iterations.
+set Max nonlinear iterations               = 100
+
+# Reducing the CFL values and/or maximum time step may improve the convergence behavior 
+# of the Stokes and thermal (advection-diffusion) systems. 
+# However, reducing these values inherently requires more time steps to reach the same end time. 
+# Finding the optimal balance frequently requires testing, although the maximum time step 
+# in lithospheric dynamics simulations with Drucker Prager plasticity is frequently 
+# not set above 20-50 thousand years. 
+set CFL number                             = 0.5
+set Maximum time step                      = 20e3
+set Output directory                       = output_oceanic_extension
+
+# Pressure normalization is turned off because using the free surface implies a surface stress of zero. 
+set Pressure normalization                 = no
+set Adiabatic surface temperature          = 1600
+
+# Governing equations 
+subsection Formulation
+  set Formulation          = Boussinesq approximation
+end
+
+# Model geometry (400x100 km)
+# To extend
```

Comment:
what does this line refer to?

Reply:
Deleted this line and added a short comment